torchvision/ops/boxes.py

```diff
@@ -4,7 +4,7 @@
 import torchvision
 
 
-@torch.jit.script
+@torch.jit._script_if_tracing
```

Comment:
I wonder if removing this `@torch.jit.script` from here (which I think is not needed) would fix the test issues?